### PR TITLE
buttons for guild name & status in PV

### PR DESCRIPTION
**Pull request by Lulonaut**
Reduces API requests by requiring the user to click a button instead of always loading the current status and guild name when PV is opened.

Depends on #762

### CODE REVIEW

**Review by romangraef — CHANGES_REQUESTED** [review]
image
the guild button overlaps with pronouns

**Review by romangraef — APPROVED** [review]

**Review by NopoTheGamer — CHANGES_REQUESTED** [review]
status and guild info is cached (i think?) but you still have to click on load for it to show

**Comment by Lulonaut** [comment]
> status and guild info is cached (i think?) but you still have to click on load for it to show

The thing that is cached is the API request and I do not feel like making changes that deep into the api request pipeline for this single feature

File: src/main/java/io/github/moulberry/notenoughupdates/profileviewer/SkyblockProfiles.java [hunk 1]
```diff
@@ -390,11 +392,13 @@ public JsonObject getOrLoadGuildInformation(Runnable runnable) {
...
 
 				if (jsonObject != null && jsonObject.has("success") && jsonObject.get("success").getAsBoolean()) {
-					if (!jsonObject.has("guild")) {
+					if (jsonObject.get("guild").isJsonNull()) {
+						isInGuild = false;
 						return null;
 					}
 
 					guildInformation = jsonObject.getAsJsonObject("guild");
+					System.out.println(guildInformation);
```
Comment (romangraef) [inline src/main/java/io/github/moulberry/notenoughupdates/profileviewer/SkyblockProfiles.java]: Imagine keeping a debug log in production

Comment (Lulonaut) [reply src/main/java/io/github/moulberry/notenoughupdates/profileviewer/SkyblockProfiles.java]: Can't imagine who would do such things

**Review by NopoTheGamer — APPROVED** [review]

**Review by romangraef — APPROVED** [review]

**Pull request merged**